packages/opentelemetry-tracer/src/instrumentation/constants.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/** General purpose constants. */
+export const constants = {
+  /** opentelemetry scope */
+  OPENTELEMETRY_SCOPE: '@opentelemetry',
+  /** Default prefix for instrumentation modules */
+  DEFAULT_PLUGIN_PACKAGE_NAME_PREFIX: 'instrumentation',
```

Comment:
Related https://github.com/open-telemetry/opentelemetry-specification/issues/30 **instrumentation vs adapter**. If everyone is agrees I am fine to change to adaptor.

Reply:
I think it should match the interface name. So if the interface is `Plugin` then this should be `plugin`. Otherwise we should rename the interface accordingly. Otherwise I don't really mind the name as long as it's consistent.

Reply:
Sure, changed to plugin.